src/Flow.cpp

```diff
@@ -88,6 +88,22 @@ Flow::Flow(NetworkInterface *_iface,
       that complains, it means that the srv_host has made a connection that triggered
       the issue and thus it must be accounted in reverse
      */
+    u_int8_t type=icmp_info->getType();
```

Comment:
please abstract logic behind port unreachable in an ICMPinfo class method. E.g, isPortUnreachable().